### PR TITLE
Repetetive random tests for Radix sort

### PR DESCRIPTION
**Pull request by N0W0RK**
**Shortly describe your proposed changes:**
New random tests that tests sort algorithm for increasing array length and increasing number size.

**Note the (sub-)exercises the tests are meant for:**

* GAD 7 RadixSort

### CODE REVIEW

**Comment by N0W0RK** [comment]
Something wwent wrong in the local merge. i should sleep

**Comment by babyygemperor** [comment]
Uh, okay, good night then

File: GAD 7 - Radixsort/tests/radix/RadixSortTester.java [hunk 1]
```diff
@@ -19,845 +20,997 @@ void printLine() {
...
-        elements = new int[]{20, 130, 153, 76, 138, 121, 188, 15, 51, 59, 168, 3, 162};
-        RadixSort.kSort(elements, 4);
-        assertArrayEquals(new int[] {20, 130, 153, 76, 138, 121, 188, 15, 51, 59, 168, 3, 162}, elements);
-
-        elements = new int[]{38, 20, 26, 62, 82, 8, 451, 139, 472, 440, 397, 387, 325, 69, 231};
-        RadixSort.kSort(elements, 3);
-        assertArrayEquals(new int[] {38, 20, 26, 62, 82, 8, 451, 139, 472, 440, 397, 387, 325, 69, 231}, elements);
+    @Nested
+    @DisplayName("Aamin's Tests")
+    class aaminsTests {
```
Comment (babyygemperor) [inline GAD 7 - Radixsort/tests/radix/RadixSortTester.java:173]: > Class names should be nouns, in mixed case with the first letter of each internal word capitalized

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Plus in general I find irrespective of the scenario that names should be capitalised.
Other than that! Everything else works like a charm, better than before

**Review by babyygemperor — COMMENTED** [review]
Apart from that, it's good to go

File: GAD 7 - Radixsort/tests/radix/RadixSortTester.java (hunk 1 above)
Comment (N0W0RK) [reply GAD 7 - Radixsort/tests/radix/RadixSortTester.java:173]: That was the pruduct of late night frutration fair point this should be capitalized.

Comment (babyygemperor) [reply GAD 7 - Radixsort/tests/radix/RadixSortTester.java]: Haha, happens to the best of us. You still did an amazing job, especially considering that you did it in the middle of night

Comment (N0W0RK) [reply GAD 7 - Radixsort/tests/radix/RadixSortTester.java]: Most of it was done before 10 pm :)

**Review by babyygemperor — DISMISSED** [review]

**Review by CrsiX — APPROVED** [review]
I would be fine with those changes as well :)

**Pull request merged**